src/graphql/wishlist.graphql

```diff
@@ -1,109 +1,141 @@
 # Fragments:
 
 fragment wishlist on Wishlist {
-  id
-  name
-  createdAt
-  lastUpdatedAt
-  productVariants {
-    totalCount
-  }
+    id
```

Comment:
Why the additional indentation?

Reply:
I did `yarn format` and i think it might have changed the indentation, otherwise this is a typo

Reply:
I would suggest you "fix" it manually and see whether `yarn format` tries to change it then.